frame/staking/src/lib.rs

```diff
@@ -3129,11 +3131,13 @@ impl<T: Config> Module<T> {
 		let elected_stashes = exposures.iter().cloned().map(|(x, _)| x).collect::<Vec<_>>();
 
 		if (elected_stashes.len() as u32) <= Self::minimum_validator_count() {
-			log!(
-				warn,
-				"chain does not have enough staking candidates to operate for era {:?}",
-				current_era,
-			);
+			if current_era != 0 {
```

Comment:
`!= 0` vs ` > 0` is the same?

so why the difference?

Reply:
I just updated it while you were putting the comment :P